### ISSUE TITLE
having to click dropdown twice

### ISSUE THREAD

**Issue by haldiggs** [kaliber5/ember-bootstrap#57]
using the 

http://kaliber5.github.io/ember-bootstrap/api/classes/Components.Dropdown.html

No matter which example I try I have to click the button twice... dropdown activates on 2nd click. Any tips for this?

**Comment by drewclauson** [comment]
Are you loading bootstrap.js?  I was doing that and had the same problem.  Removed bootstrap.js from my HTML HEAD and it was fixed.

**Comment by haldiggs** [comment]
hmm I am. BUT... my Ember app is only part of a larger app (shell) which needs bootstrap.js as well. This is my biggest complaint with Ember. The constant struggle :-) 

I thought this add-on would allow my own bootstrap versions (fonts, themes etc.). I used the switches to turn off the bootstrap... 

  	'ember-bootstrap': {
  		'importBootstrapCSS': false,
  		'importBootstrapFont': false
  	}

**Comment by haldiggs** [comment]
I didn't notice the additional bootstrap had been added to the bower file. hmmm. I think the add-on should only do that, be an add-on. Not control the bootstrap versions. I wonder how I can fix this.

**Comment by simonihmig** [comment]
You can control the bootstrap versions using your `bower.json`. And btw, that's what most addons are used for, including third party assets, so this is perfectly ok. 

ember-bootstrap does not rely on the original bootstrap.js, it uses native ember components for all the JavaScript functionality. That's what it is made for. In fact if you exclude the other bootstrap imports like you did, absolutely nothing from the bower installed bootstrap is included in your ember app. However if you include bootrstrap.js by yourself, which adds its own JS functionality on existing ember component controlled DOM elements, thinks go wrong. No way around this!

**Comment by haldiggs** [comment]
Appreciate the convo, the thing is, it was the add-on that added bootstrap to my ember project. Other add-ons like ember-breadcrumbs require its presence in order to function but I am able to load it from my web sites outer shell. I can confirm that this add-on did add bootstrap to my bower file.

**Comment by bitsoflogic** [comment]
Thanks for the back-n-forth on this.  I was able to resolve the problem after reading this thread.

My solution was to head over to getbootstrap.com and customize a bootstrap.min.js file.  In my case, I unchecked all Common CSS (which deselected a few components), then I unchecked JavaScript components: Dropdown and the jQuery component Dropdown.

I believe unchecking both Dropdowns and compiling it were the two critical steps.  Bootstrap 3 for me.

All my other addons are now working great with this one.
